cli/commands/log.go

```diff
@@ -0,0 +1,77 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+
+	"github.com/tarantool/cartridge-cli/cli/running"
+)
+
+var (
+	followLog bool
+	linesLog  int
+
+	linesFlagDoc = fmt.Sprintf(
+		"Number of lines to output (from the end).\nDefaults to %d\n", defaultLinesLog,
+	)
+)
+
+const (
+	defaultLinesLog = 15
+)
+
+func init() {
+	rootCmd.AddCommand(logCmd)
+
+	logCmd.Flags().StringVar(&projectCtx.Name, "name", "", nameFlagDoc)
+
+	logCmd.Flags().StringVar(&projectCtx.RunDir, "run-dir", "", runDirFlagDoc)
+	logCmd.Flags().StringVar(&projectCtx.LogDir, "log-dir", "", logDirFlagDoc)
+	logCmd.Flags().StringVar(&projectCtx.ConfPath, "cfg", "", cfgFlagDoc)
+
+	logCmd.Flags().BoolVar(&projectCtx.WithStateboard, "stateboard", false, stateboardFlagDoc)
+	logCmd.Flags().BoolVar(&projectCtx.StateboardOnly, "stateboard-only", false, stateboardOnlyFlagDoc)
+
+	logCmd.Flags().BoolVarP(&followLog, "follow", "f", false, followFlagDoc)
+	logCmd.Flags().IntVarP(&linesLog, "lines", "n", 0, linesFlagDoc)
+}
+
+var logCmd = &cobra.Command{
+	Use:   "log [INSTANCE_NAME...]",
+	Short: "Get logs of instance(s)",
+	Long:  fmt.Sprintf("Get logs of instance(s)\n\n%s", runningCommonDoc),
+	Run: func(cmd *cobra.Command, args []string) {
+		err := runLogCmd(cmd, args)
+		if err != nil {
+			log.Fatalf(err.Error())
+		}
+	},
+}
+
+func runLogCmd(cmd *cobra.Command, args []string) error {
+	linesFlag := cmd.Flags().Lookup("lines")
+	if linesFlag == nil {
+		return fmt.Errorf("Failed to find lines flag")
+	}
+
+	if !linesFlag.Changed {
+		linesLog = defaultLinesLog
+	}
+
+	if err := running.FillCtx(&projectCtx, args); err != nil {
+		return err
+	}
+
+	if err := running.Log(&projectCtx, followLog, linesLog); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+const (
+	followFlagDoc = `Output appended data as the log grows
```

Comment:
`follow` description from `man journalctl`.
```
Show only the most recent journal entries, and continuously print new entries as they are appended to the journal.
```

Reply:
I was thinking about `tail` command writing this feature. Why do you think that current description is bad?

Reply:
> Why do you think that current description is bad?

I don't. But I suggest you an alternative.